mesonbuild/interpreter.py

```diff
@@ -2470,9 +2465,11 @@ def do_subproject(self, dirname, kwargs):
         return self.subprojects[dirname]
 
     def get_option_internal(self, optname):
+        # Some base options are not defined in some environments, return the
+        # default value from compilers.base_options in that case.
         for d in chain(
-                [self.coredata.base_options, compilers.base_options, self.coredata.builtins],
-                self.coredata.get_all_compiler_options()):
+                  [self.coredata.base_options, compilers.base_options,
```

Comment:
__[Flake8]__ [E126] continuation line over-indented for hanging indent ([link](https://sider.review/gh/repos/19784232/pulls/5253#issue-3978663))

<sub>posted by Sider</sub>